Ops/src/OpsWorker.cpp

```diff
@@ -364,7 +365,11 @@ size_t Worker::getCurrentNumberOfIterations()
 
 double Worker::opsRand()
 {
-    return double(rand())/double(RAND_MAX);
+    std::random_device device;
+    std::mt19937 generator(device());
```

Comment:
Any particular reason for using the mt1993 generator? (I dont know anything about these things), but there is also:
`std::default_random_engine generator()`, slightly simpler code, but does not really matter.

Reply:
I changed it do the default engine. I got the code from a suggestion on some forum. I have no idea which one is best, but for our purposes they should all be good enough I think.